### ISSUE TITLE
Permission denied when pulling private image from quay.io

### ISSUE THREAD

**Issue by Knovour** [containrrr/watchtower#6]
Hi, I want to check quay.io private repo update by watchtower.

But I always got

```
Error: Status 403 trying to pull repository private/repo-name: "{\"error\": \"Permission Denied\"}"
```

OS: Ubuntu 15.04
Docker version: 1.9.1

Any solution?

**Comment by hadronzoo** [comment]
I'm also seeing this error.

**Comment by codeocelot** [comment]
+1

**Comment by radekdymacz** [comment]
+1

**Comment by matt-land** [comment]
+1

**Comment by chrisferry** [comment]
👍

**Comment by kevinresol** [comment]
+1

**Comment by ghost** [comment]
+1. I see this error pulling for artifactory as well. I suspect this will happen with any private docker registry

**Comment by ghost** [comment]
I have validated that running the same GET command I get the same error until I add a basciAuth token for the registry. My thoughts here are that the provided auth username/password are not correctly being converted to a basic auth token (base64) or not provided at all during the Get request.

**Comment by TheDeadCode** [comment]
+1. Fails on self-hosted docker too.

**Comment by stffabi** [comment]
New official builds are now available on [docker hub](https://hub.docker.com/r/v2tec/watchtower/tags/). Issue should be fixed with v0.1.0
